t/05-scheduler-full.t

```diff
@@ -277,17 +277,18 @@ subtest 'Simulation of heavy unstable load' => sub {
 
 subtest 'Websocket server - close connection test' => sub {
     kill_service($wspid);
-    my $log_file        = tempfile;
+    my $log_file = tempfile;
+    local $ENV{OPENQA_LOGFILE};
+    local $ENV{MOJO_LOG_LEVEL};
     my $unstable_ws_pid = create_websocket_server($mojoport + 1, 1);
-    my $w2_pid          = create_worker($k->key, $k->secret, "http://localhost:$mojoport", 2, $log_file);
-    my $re              = qr/\[.*?\]\s\[.*?\]\sConnection turned off from .*?\- (.*?)\s\:(.*?) dead/;
+    my $w2_pid = create_worker($k->key, $k->secret, "http://localhost:$mojoport", 2, $log_file);
```

Comment:
I wonder how would it behave if the same logfile is used for all the workers?... (I guess ends up in an error?)?

Reply:
That's a good test. I think mojolicious will append the several outputs to the same file...
I'll create a small test.